package.json

```diff
@@ -79,7 +79,7 @@
     "husky": "^0.13.3",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^18.0.0",
-    "lerna": "2.0.0-beta.36",
+    "lerna": "^2.0.0-rc.4",
```

Comment:
I would update this to 
```
"lerna": "2.0.0-rc.4",
```
to ensure our script and lerna.json file are in sync with the installed version